tools/pybind11Tools.cmake

```diff
@@ -0,0 +1,140 @@
+# tools/pybind11Tools.cmake -- Build system for the pybind11 modules
+#
+# Copyright (c) 2015 Wenzel Jakob <wenzel@inf.ethz.ch>
+#
+# All rights reserved. Use of this source code is governed by a
+# BSD-style license that can be found in the LICENSE file.
+
+cmake_minimum_required(VERSION 2.8.12)
+
+# Add a CMake parameter for choosing a desired Python version
+set(PYBIND11_PYTHON_VERSION "" CACHE STRING "Python version to use for compiling modules")
+
+set(Python_ADDITIONAL_VERSIONS 3.4 3.5 3.6 3.7)
```

Comment:
The variable is still needed because `FindPythonLibsNew` calls `find_package(PythonInterp)` and on older CMake version `FindPythonInterp` does cannot find new Python 3.x executables (e.g. CMake 2.8.12 only checks up to 3.3).